package/openocd_cypress/openocd_cypress.mk

```diff
@@ -25,6 +25,8 @@ OPENOCD_CYPRESS_CONF_OPTS += --disable-shared
 OPENOCD_CYPRESS_CONF_OPTS += --enable-dummy
 OPENOCD_CYPRESS_CONF_OPTS += --disable-werror
 
```

Comment:
These files might be targeting 32-bit ARM, but I believe they are download as-is to the target.  This should preclude inclusion.